src/helics/core/FederateState.cpp

```diff
@@ -1087,6 +1054,9 @@ iteration_state FederateState::processActionMessage (ActionMessage &cmd)
             return iteration_state::next_step;
         }
         break;
+    case CMD_FED_CONFIGURE:
+        processConfigUpdate(cmd);
+        break;
```

Comment:
Shouldn't we add a `default` case here?

Reply:
there is one, it is just at the top of the function
```
case CMD_IGNORE:
    default:
        break;
```